error: redefinition of class std::numeric_limits<__float128>

Hi, according to https://gcc.gnu.org/gcc-14/changes.html with gcc (GCC) 14.1.1 20240507 or later.

> std::numeric_limits<_Float32> and std::numeric_limits<_Float64> are now defined for all standard modes, not only for C++23.

## `opm-common-2023.10`

<details>
  <summary>CMake log</summary>

  ```console
  -- The C compiler identification is GNU 14.1.1
  -- The CXX compiler identification is GNU 14.1.1
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /usr/sbin/gcc - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /usr/sbin/g++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  CMake Warning at cmake/Modules/Findopm-tests.cmake:26 (message):
    opm-tests not found - integration tests using opm-tests will be skipped.
  Call Stack (most recent call first):
    CMakeLists.txt:62 (include)
  -- Performing Test HAVE_C99
  -- Performing Test HAVE_C99 - Success
  -- Found C99: -std=c99
  -- Could NOT find Valgrind (missing: VALGRIND_INCLUDE_DIR VALGRIND_PROGRAM) 
  -- Found Boost: /usr/lib/cmake/Boost-1.83.0/BoostConfig.cmake (found suitable version "1.83.0", minimum required is "1.44.0") found components: system unit_test_framework
  -- Performing Test HAVE_CJSON
  -- Performing Test HAVE_CJSON - Success
  -- Found cjson: /usr/include/cjson
  -- Performing Test QuadMath_COMPILES
  -- Performing Test QuadMath_COMPILES - Success
  -- Found QuadMath: (Supported by compiler)
  -- Found Python3: /usr/sbin/python3.12 (found version "3.12.3") found components: Interpreter Development.Module
  -- Performing Test HAS_FLTO
  -- Performing Test HAS_FLTO - Success
  -- Found pybind11: /usr/include (found version "2.12.0")
  -- Performing Test HAVE_ATTRIBUTE_ALWAYS_INLINE
  -- Performing Test HAVE_ATTRIBUTE_ALWAYS_INLINE - Success
  -- Performing Test HAS_ATTRIBUTE_UNUSED
  -- Performing Test HAS_ATTRIBUTE_UNUSED - Success
  -- Performing Test HAS_ATTRIBUTE_DEPRECATED
  -- Performing Test HAS_ATTRIBUTE_DEPRECATED - Success
  -- Performing Test HAS_ATTRIBUTE_DEPRECATED_MSG
  -- Performing Test HAS_ATTRIBUTE_DEPRECATED_MSG - Success
  -- CMake version: 3.29.3
  -- Linux distribution: Arch Linux
  -- Target architecture: x86_64
  -- Found Git: /usr/sbin/git (found version "2.45.0")
  -- Source code repository: not found!
  -- GNU C++ compiler version: 14.1.1 20240507
  -- Linker: ld 2.42.0
  Processing opm_defaults opm-common
  -- Checking to see if CXX compiler accepts flag -Wl,--enable-new-dtags
  -- Checking to see if CXX compiler accepts flag -Wl,--enable-new-dtags - yes
  -- Precompiled headers: disabled
  -- Build type: None
  -- Checking to see if CXX compiler accepts flag -mtune=native
  -- Checking to see if CXX compiler accepts flag -mtune=native - yes
  -- Found OpenMP_C: -fopenmp (found version "4.5")
  -- Found OpenMP_CXX: -fopenmp (found version "4.5")
  -- Found OpenMP: TRUE (found version "4.5")
  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
  -- Found Threads: TRUE
  -- Checking to see if CXX compiler accepts flag -pthread
  -- Checking to see if CXX compiler accepts flag -pthread - yes
  -- Could NOT find CppCheck (missing: CPPCHECK_PROGRAM) 
  -- Disabling clang-check as CMAKE_EXPORT_COMPILE_COMMANDS is not enabled
  -- Could NOT find Valgrind (missing: VALGRIND_INCLUDE_DIR VALGRIND_PROGRAM) 
  -- Generating debug symbols: -ggdb3
  -- Looking for strip utility
  -- Looking for strip utility - found
  -- Performing Test HAVE_DYNAMIC_BOOST_TEST
  -- Performing Test HAVE_DYNAMIC_BOOST_TEST - Success
  -- Writing config file "/tmp/makepkg/opm-common/src/build-cmake/config.h"...
  -- This build defaults to installing in /usr
  -- Checking whether the file system is case-sensitive
  -- File system is case-sensitive
  -- Found Doxygen: /usr/sbin/doxygen (found version "1.10.0") found components: doxygen dot
  -- Writing version information to local header project-version.h
  -- Looking for sgemm_
  -- Looking for sgemm_ - not found
  -- Looking for sgemm_
  -- Looking for sgemm_ - found
  -- Found BLAS: /usr/lib/libblas.so
  -- Looking for cheev_
  -- Looking for cheev_ - not found
  -- Looking for cheev_
  -- Looking for cheev_ - found
  -- Found LAPACK: /usr/lib/liblapack.so;/usr/lib/libblas.so
  -- Performing Test HAVE_CXA_DEMANGLE
  -- Performing Test HAVE_CXA_DEMANGLE - Success
  -- Found MPI_C: /usr/lib/libmpi.so (found version "3.1")
  -- Found MPI_CXX: /usr/lib/libmpi.so (found version "3.1")
  -- Found MPI: TRUE (found version "3.1")
  -- Found GMP: /usr/lib/libgmpxx.so
  -- Found dune-common: /usr/include
  -- Looking for MPI_Finalized
  -- Looking for MPI_Finalized - found
  -- Version 2.9.1 of dune-common from /usr/lib/cmake/dune-common
  -- Configuring done (6.7s)
  -- Generating done (0.6s)
  -- Build files have been written to: /tmp/makepkg/opm-common/src/build-cmake
  [  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/json/JsonObject.cpp.o
  [  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/createDefaultKeywordList.cpp.o
  [  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/DeckValue.cpp.o
  [  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/UDAValue.cpp.o
  [  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/DeckTree.cpp.o
  [  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/Deck.cpp.o
  [  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/DeckView.cpp.o
  [  0%] Built target copy_python
  [  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/DeckItem.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/DeckKeyword.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/DeckRecord.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Deck/DeckOutput.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Generator/KeywordGenerator.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Generator/KeywordLoader.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Schedule/UDQ/UDQEnums.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/ErrorGuard.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/ParseContext.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/ParserEnums.cpp.o
  [  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/ParserItem.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/ParserKeyword.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/ParserRecord.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/raw/RawKeyword.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/raw/RawRecord.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Parser/raw/StarToken.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Units/Dimension.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/input/eclipse/Units/UnitSystem.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/utility/OpmInputError.cpp.o
  [  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/utility/shmatch.cpp.o
  [  6%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/utility/String.cpp.o
  [  6%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/OpmLog.cpp.o
  [  6%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/Logger.cpp.o
  [  6%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/StreamLog.cpp.o
  [  6%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/LogBackend.cpp.o
  [  6%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/LogUtil.cpp.o
  [  6%] Linking CXX executable bin/genkw
  [  6%] Built target genkw
  [  6%] Generating tmp_gen/ParserKeywords/A.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/A.hpp, tmp_gen/ParserKeywords/ParserInitA.cpp, tmp_gen/ParserKeywords/BuiltinA.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitA.hpp, tmp_gen/ParserKeywords/B.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/B.hpp, tmp_gen/ParserKeywords/ParserInitB.cpp, tmp_gen/ParserKeywords/BuiltinB.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitB.hpp, tmp_gen/ParserKeywords/C.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/C.hpp, tmp_gen/ParserKeywords/ParserInitC.cpp, tmp_gen/ParserKeywords/BuiltinC.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitC.hpp, tmp_gen/ParserKeywords/D.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/D.hpp, tmp_gen/ParserKeywords/ParserInitD.cpp, tmp_gen/ParserKeywords/BuiltinD.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitD.hpp, tmp_gen/ParserKeywords/E.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/E.hpp, tmp_gen/ParserKeywords/ParserInitE.cpp, tmp_gen/ParserKeywords/BuiltinE.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitE.hpp, tmp_gen/ParserKeywords/F.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/F.hpp, tmp_gen/ParserKeywords/ParserInitF.cpp, tmp_gen/ParserKeywords/BuiltinF.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitF.hpp, tmp_gen/ParserKeywords/G.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/G.hpp, tmp_gen/ParserKeywords/ParserInitG.cpp, tmp_gen/ParserKeywords/BuiltinG.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitG.hpp, tmp_gen/ParserKeywords/H.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/H.hpp, tmp_gen/ParserKeywords/ParserInitH.cpp, tmp_gen/ParserKeywords/BuiltinH.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitH.hpp, tmp_gen/ParserKeywords/I.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/I.hpp, tmp_gen/ParserKeywords/ParserInitI.cpp, tmp_gen/ParserKeywords/BuiltinI.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitI.hpp, tmp_gen/ParserKeywords/J.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/J.hpp, tmp_gen/ParserKeywords/ParserInitJ.cpp, tmp_gen/ParserKeywords/BuiltinJ.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitJ.hpp, tmp_gen/ParserKeywords/K.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/K.hpp, tmp_gen/ParserKeywords/ParserInitK.cpp, tmp_gen/ParserKeywords/BuiltinK.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitK.hpp, tmp_gen/ParserKeywords/L.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/L.hpp, tmp_gen/ParserKeywords/ParserInitL.cpp, tmp_gen/ParserKeywords/BuiltinL.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitL.hpp, tmp_gen/ParserKeywords/M.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/M.hpp, tmp_gen/ParserKeywords/ParserInitM.cpp, tmp_gen/ParserKeywords/BuiltinM.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitM.hpp, tmp_gen/ParserKeywords/N.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/N.hpp, tmp_gen/ParserKeywords/ParserInitN.cpp, tmp_gen/ParserKeywords/BuiltinN.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitN.hpp, tmp_gen/ParserKeywords/O.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/O.hpp, tmp_gen/ParserKeywords/ParserInitO.cpp, tmp_gen/ParserKeywords/BuiltinO.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitO.hpp, tmp_gen/ParserKeywords/P.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/P.hpp, tmp_gen/ParserKeywords/ParserInitP.cpp, tmp_gen/ParserKeywords/BuiltinP.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitP.hpp, tmp_gen/ParserKeywords/Q.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/Q.hpp, tmp_gen/ParserKeywords/ParserInitQ.cpp, tmp_gen/ParserKeywords/BuiltinQ.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitQ.hpp, tmp_gen/ParserKeywords/R.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/R.hpp, tmp_gen/ParserKeywords/ParserInitR.cpp, tmp_gen/ParserKeywords/BuiltinR.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitR.hpp, tmp_gen/ParserKeywords/S.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/S.hpp, tmp_gen/ParserKeywords/ParserInitS.cpp, tmp_gen/ParserKeywords/BuiltinS.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitS.hpp, tmp_gen/ParserKeywords/T.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/T.hpp, tmp_gen/ParserKeywords/ParserInitT.cpp, tmp_gen/ParserKeywords/BuiltinT.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitT.hpp, tmp_gen/ParserKeywords/U.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/U.hpp, tmp_gen/ParserKeywords/ParserInitU.cpp, tmp_gen/ParserKeywords/BuiltinU.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitU.hpp, tmp_gen/ParserKeywords/V.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/V.hpp, tmp_gen/ParserKeywords/ParserInitV.cpp, tmp_gen/ParserKeywords/BuiltinV.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitV.hpp, tmp_gen/ParserKeywords/W.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/W.hpp, tmp_gen/ParserKeywords/ParserInitW.cpp, tmp_gen/ParserKeywords/BuiltinW.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitW.hpp, tmp_gen/ParserKeywords/X.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/X.hpp, tmp_gen/ParserKeywords/ParserInitX.cpp, tmp_gen/ParserKeywords/BuiltinX.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitX.hpp, tmp_gen/ParserKeywords/Y.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/Y.hpp, tmp_gen/ParserKeywords/ParserInitY.cpp, tmp_gen/ParserKeywords/BuiltinY.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitY.hpp, tmp_gen/ParserKeywords/Z.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/Z.hpp, tmp_gen/ParserKeywords/ParserInitZ.cpp, tmp_gen/ParserKeywords/BuiltinZ.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitZ.hpp, tmp_gen/TestKeywords.cpp, tmp_gen/ParserInit.cpp, tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/Builtin.hpp, tmp_gen/builtin_pybind11.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/A.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/B.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/C.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/D.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/E.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/F.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/G.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/H.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/I.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/J.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/K.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/L.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/M.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/N.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/O.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/P.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/Q.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/R.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/S.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/T.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/U.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/V.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/W.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/X.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/Y.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/Z.cpp
  Updated init header for A file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitA.hpp
  Updated init source for A file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitA.cpp
  Updated init header for B file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitB.hpp
  Updated init source for B file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitB.cpp
  Updated init header for C file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitC.hpp
  Updated init source for C file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitC.cpp
  Updated init header for D file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitD.hpp
  Updated init source for D file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitD.cpp
  Updated init header for E file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitE.hpp
  Updated init source for E file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitE.cpp
  Updated init header for F file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitF.hpp
  Updated init source for F file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitF.cpp
  Updated init header for G file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitG.hpp
  Updated init source for G file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitG.cpp
  Updated init header for H file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitH.hpp
  Updated init source for H file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitH.cpp
  Updated init header for I file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitI.hpp
  Updated init source for I file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitI.cpp
  Updated init header for J file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitJ.hpp
  Updated init source for J file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitJ.cpp
  Updated init header for K file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitK.hpp
  Updated init source for K file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitK.cpp
  Updated init header for L file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitL.hpp
  Updated init source for L file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitL.cpp
  Updated init header for M file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitM.hpp
  Updated init source for M file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitM.cpp
  Updated init header for N file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitN.hpp
  Updated init source for N file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitN.cpp
  Updated init header for O file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitO.hpp
  Updated init source for O file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitO.cpp
  Updated init header for P file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitP.hpp
  Updated init source for P file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitP.cpp
  Updated init header for Q file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitQ.hpp
  Updated init source for Q file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitQ.cpp
  Updated init header for R file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitR.hpp
  Updated init source for R file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitR.cpp
  Updated init header for S file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitS.hpp
  Updated init source for S file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitS.cpp
  Updated init header for T file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitT.hpp
  Updated init source for T file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitT.cpp
  Updated init header for U file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitU.hpp
  Updated init source for U file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitU.cpp
  Updated init header for V file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitV.hpp
  Updated init source for V file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitV.cpp
  Updated init header for W file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitW.hpp
  Updated init source for W file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitW.cpp
  Updated init header for X file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitX.hpp
  Updated init source for X file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitX.cpp
  Updated init header for Y file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitY.hpp
  Updated init source for Y file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitY.cpp
  Updated init header for Z file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/ParserInitZ.hpp
  Updated init source for Z file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/ParserInitZ.cpp
  Updated init file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserInit.cpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/A.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/B.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/C.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/D.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/E.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/F.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/G.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/H.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/I.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/J.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/K.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/L.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/M.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/N.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/O.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/P.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/Q.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/R.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/S.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/T.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/U.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/V.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/W.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/X.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/Y.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/Z.hpp
  Updated header file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/include/opm/input/eclipse/Parser/ParserKeywords/Builtin.hpp
  Updated builtin source for A file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinA.cpp
  Updated builtin source for B file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinB.cpp
  Updated builtin source for C file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinC.cpp
  Updated builtin source for D file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinD.cpp
  Updated builtin source for E file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinE.cpp
  Updated builtin source for F file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinF.cpp
  Updated builtin source for G file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinG.cpp
  Updated builtin source for H file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinH.cpp
  Updated builtin source for I file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinI.cpp
  Updated builtin source for J file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinJ.cpp
  Updated builtin source for K file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinK.cpp
  Updated builtin source for L file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinL.cpp
  Updated builtin source for M file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinM.cpp
  Updated builtin source for N file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinN.cpp
  Updated builtin source for O file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinO.cpp
  Updated builtin source for P file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinP.cpp
  Updated builtin source for Q file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinQ.cpp
  Updated builtin source for R file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinR.cpp
  Updated builtin source for S file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinS.cpp
  Updated builtin source for T file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinT.cpp
  Updated builtin source for U file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinU.cpp
  Updated builtin source for V file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinV.cpp
  Updated builtin source for W file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinW.cpp
  Updated builtin source for X file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinX.cpp
  Updated builtin source for Y file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinY.cpp
  Updated builtin source for Z file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/ParserKeywords/BuiltinZ.cpp
  Writing file: /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/builtin_pybind11.cpp
  Updated source file written to /tmp/makepkg/opm-common/src/build-cmake/tmp_gen/builtin_pybind11.cpp
  [  6%] Generating ParserKeywords/A.cpp, include/opm/input/eclipse/Parser/ParserKeywords/A.hpp, ParserKeywords/ParserInitA.cpp, ParserKeywords/BuiltinA.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitA.hpp, ParserKeywords/B.cpp, include/opm/input/eclipse/Parser/ParserKeywords/B.hpp, ParserKeywords/ParserInitB.cpp, ParserKeywords/BuiltinB.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitB.hpp, ParserKeywords/C.cpp, include/opm/input/eclipse/Parser/ParserKeywords/C.hpp, ParserKeywords/ParserInitC.cpp, ParserKeywords/BuiltinC.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitC.hpp, ParserKeywords/D.cpp, include/opm/input/eclipse/Parser/ParserKeywords/D.hpp, ParserKeywords/ParserInitD.cpp, ParserKeywords/BuiltinD.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitD.hpp, ParserKeywords/E.cpp, include/opm/input/eclipse/Parser/ParserKeywords/E.hpp, ParserKeywords/ParserInitE.cpp, ParserKeywords/BuiltinE.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitE.hpp, ParserKeywords/F.cpp, include/opm/input/eclipse/Parser/ParserKeywords/F.hpp, ParserKeywords/ParserInitF.cpp, ParserKeywords/BuiltinF.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitF.hpp, ParserKeywords/G.cpp, include/opm/input/eclipse/Parser/ParserKeywords/G.hpp, ParserKeywords/ParserInitG.cpp, ParserKeywords/BuiltinG.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitG.hpp, ParserKeywords/H.cpp, include/opm/input/eclipse/Parser/ParserKeywords/H.hpp, ParserKeywords/ParserInitH.cpp, ParserKeywords/BuiltinH.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitH.hpp, ParserKeywords/I.cpp, include/opm/input/eclipse/Parser/ParserKeywords/I.hpp, ParserKeywords/ParserInitI.cpp, ParserKeywords/BuiltinI.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitI.hpp, ParserKeywords/J.cpp, include/opm/input/eclipse/Parser/ParserKeywords/J.hpp, ParserKeywords/ParserInitJ.cpp, ParserKeywords/BuiltinJ.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitJ.hpp, ParserKeywords/K.cpp, include/opm/input/eclipse/Parser/ParserKeywords/K.hpp, ParserKeywords/ParserInitK.cpp, ParserKeywords/BuiltinK.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitK.hpp, ParserKeywords/L.cpp, include/opm/input/eclipse/Parser/ParserKeywords/L.hpp, ParserKeywords/ParserInitL.cpp, ParserKeywords/BuiltinL.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitL.hpp, ParserKeywords/M.cpp, include/opm/input/eclipse/Parser/ParserKeywords/M.hpp, ParserKeywords/ParserInitM.cpp, ParserKeywords/BuiltinM.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitM.hpp, ParserKeywords/N.cpp, include/opm/input/eclipse/Parser/ParserKeywords/N.hpp, ParserKeywords/ParserInitN.cpp, ParserKeywords/BuiltinN.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitN.hpp, ParserKeywords/O.cpp, include/opm/input/eclipse/Parser/ParserKeywords/O.hpp, ParserKeywords/ParserInitO.cpp, ParserKeywords/BuiltinO.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitO.hpp, ParserKeywords/P.cpp, include/opm/input/eclipse/Parser/ParserKeywords/P.hpp, ParserKeywords/ParserInitP.cpp, ParserKeywords/BuiltinP.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitP.hpp, ParserKeywords/Q.cpp, include/opm/input/eclipse/Parser/ParserKeywords/Q.hpp, ParserKeywords/ParserInitQ.cpp, ParserKeywords/BuiltinQ.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitQ.hpp, ParserKeywords/R.cpp, include/opm/input/eclipse/Parser/ParserKeywords/R.hpp, ParserKeywords/ParserInitR.cpp, ParserKeywords/BuiltinR.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitR.hpp, ParserKeywords/S.cpp, include/opm/input/eclipse/Parser/ParserKeywords/S.hpp, ParserKeywords/ParserInitS.cpp, ParserKeywords/BuiltinS.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitS.hpp, ParserKeywords/T.cpp, include/opm/input/eclipse/Parser/ParserKeywords/T.hpp, ParserKeywords/ParserInitT.cpp, ParserKeywords/BuiltinT.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitT.hpp, ParserKeywords/U.cpp, include/opm/input/eclipse/Parser/ParserKeywords/U.hpp, ParserKeywords/ParserInitU.cpp, ParserKeywords/BuiltinU.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitU.hpp, ParserKeywords/V.cpp, include/opm/input/eclipse/Parser/ParserKeywords/V.hpp, ParserKeywords/ParserInitV.cpp, ParserKeywords/BuiltinV.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitV.hpp, ParserKeywords/W.cpp, include/opm/input/eclipse/Parser/ParserKeywords/W.hpp, ParserKeywords/ParserInitW.cpp, ParserKeywords/BuiltinW.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitW.hpp, ParserKeywords/X.cpp, include/opm/input/eclipse/Parser/ParserKeywords/X.hpp, ParserKeywords/ParserInitX.cpp, ParserKeywords/BuiltinX.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitX.hpp, ParserKeywords/Y.cpp, include/opm/input/eclipse/Parser/ParserKeywords/Y.hpp, ParserKeywords/ParserInitY.cpp, ParserKeywords/BuiltinY.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitY.hpp, ParserKeywords/Z.cpp, include/opm/input/eclipse/Parser/ParserKeywords/Z.hpp, ParserKeywords/ParserInitZ.cpp, ParserKeywords/BuiltinZ.cpp, include/opm/input/eclipse/Parser/ParserKeywords/ParserInitZ.hpp, TestKeywords.cpp, ParserInit.cpp, include/opm/input/eclipse/Parser/ParserKeywords/Builtin.hpp, python/cxx/builtin_pybind11.cpp
  [  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/Z.cpp.o
  [  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitZ.cpp.o
  [  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinZ.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitY.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinX.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/Y.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinY.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitX.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/X.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinW.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitW.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/W.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinV.cpp.o
  [  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitV.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/V.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinU.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitU.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/U.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinT.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitT.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/T.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinS.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitS.cpp.o
  [ 10%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/S.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinR.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitR.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/R.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinQ.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitQ.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/Q.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinP.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitP.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/P.cpp.o
  [ 12%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinO.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitO.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/O.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinN.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitN.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/N.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinM.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitM.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/M.cpp.o
  [ 14%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinL.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitL.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/L.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinK.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitK.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/K.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinJ.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitJ.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/J.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinI.cpp.o
  [ 16%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitI.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/I.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinH.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitH.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/H.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinG.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitG.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/G.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinF.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitF.cpp.o
  [ 18%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/F.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinE.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitE.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/E.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinD.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitD.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/D.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinC.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitC.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/C.cpp.o
  [ 20%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinB.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitB.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/B.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/BuiltinA.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/ParserInitA.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/A.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/ParserInit.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/CounterLog.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/EclipsePRTLog.cpp.o
  [ 22%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/LogBackend.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/Logger.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/LogUtil.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/KeywordLocation.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/InfoLogger.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/OpmLog.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/StreamLog.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/TimerLog.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/ActiveGridCells.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/Demangle.cpp.o
  [ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/FileSystem.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/MemPacker.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/numeric/MonotCubicInterpolator.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/OpmInputError.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/parameters/Parameter.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/parameters/ParameterGroup.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/parameters/ParameterRequirement.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/parameters/ParameterTools.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/numeric/calculateCellVol.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/numeric/RootFinders.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/shmatch.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/String.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/TimeService.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/common/Spline.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/common/Tabulated1DFunction.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/common/TridiagonalMatrix.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/common/UniformXTabulated2DFunction.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/components/CO2.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/components/H2.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/densead/Evaluation.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/fluidmatrixinteractions/EclEpsScalingPoints.cpp.o
  [ 30%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/fluidsystems/BlackOilFluidSystem.cpp.o
  [ 30%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/fluidsystems/blackoilpvt/DeadOilPvt.cpp.o
  [ 30%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/material/fluidsystems/blackoilpvt/DryGasPvt.cpp.o
  In file included from /tmp/makepkg/opm-common/src/opm-common-release-2023.10-final/src/opm/material/densead/Evaluation.cpp:28:
  /tmp/makepkg/opm-common/src/opm-common-release-2023.10-final/opm/material/common/quad.hpp:50:7: error: redefinition of ‘class std::numeric_limits<__float128>’
     50 | class numeric_limits<quad>
        |       ^~~~~~~~~~~~~~~~~~~~
  In file included from /usr/include/c++/14.1.1/bits/specfun.h:44,
                   from /usr/include/c++/14.1.1/cmath:3898,
                   from /tmp/makepkg/opm-common/src/opm-common-release-2023.10-final/opm/material/common/quad.hpp:32:
  /usr/include/c++/14.1.1/limits:2089:12: note: previous definition of ‘class std::numeric_limits<__float128>’
   2089 |     struct numeric_limits<__float128>
        |            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  In file included from /tmp/makepkg/opm-common/src/opm-common-release-2023.10-final/src/opm/material/components/CO2.cpp:27:
  /tmp/makepkg/opm-common/src/opm-common-release-2023.10-final/opm/material/common/quad.hpp:50:7: error: redefinition of ‘class std::numeric_limits<__float128>’
     50 | class numeric_limits<quad>
        |       ^~~~~~~~~~~~~~~~~~~~
  In file included from /usr/include/c++/14.1.1/bits/specfun.h:44,
                   from /usr/include/c++/14.1.1/cmath:3898,
                   from /tmp/makepkg/opm-common/src/opm-common-release-2023.10-final/opm/material/Constants.hpp:30,
                   from /tmp/makepkg/opm-common/src/opm-common-release-2023.10-final/opm/material/components/CO2.hpp:31,
                   from /tmp/makepkg/opm-common/src/opm-common-release-2023.10-final/src/opm/material/components/CO2.cpp:25:
  /usr/include/c++/14.1.1/limits:2089:12: note: previous definition of ‘class std::numeric_limits<__float128>’
   2089 |     struct numeric_limits<__float128>
        |            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  make[3]: *** [CMakeFiles/opmcommon.dir/build.make:3712: CMakeFiles/opmcommon.dir/src/opm/material/densead/Evaluation.cpp.o] Error 1
  make[3]: *** Waiting for unfinished jobs....
  make[3]: *** [CMakeFiles/opmcommon.dir/build.make:3684: CMakeFiles/opmcommon.dir/src/opm/material/components/CO2.cpp.o] Error 1
  make[2]: *** [CMakeFiles/Makefile2:529: CMakeFiles/opmcommon.dir/all] Error 2
  make[1]: *** [CMakeFiles/Makefile2:6784: CMakeFiles/opmcommon_python.dir/rule] Error 2
  make: *** [Makefile:3143: opmcommon_python] Error 2
  ```
</details>

## `opm-common-2024.04`

<details>
  <summary>CMake log</summary>

  ```console
  -- The C compiler identification is GNU 14.1.1
  -- The CXX compiler identification is GNU 14.1.1
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /usr/sbin/gcc - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /usr/sbin/g++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  CMake Warning at cmake/Modules/Findopm-tests.cmake:26 (message):
    opm-tests not found - integration tests using opm-tests will be skipped.
  Call Stack (most recent call first):
    CMakeLists.txt:62 (include)
  -- Performing Test HAVE_C99
  -- Performing Test HAVE_C99 - Success
  -- Found C99: -std=c99
  -- Could NOT find Valgrind (missing: VALGRIND_INCLUDE_DIR VALGRIND_PROGRAM) 
  -- Found Boost: /usr/lib/cmake/Boost-1.83.0/BoostConfig.cmake (found suitable version "1.83.0", minimum required is "1.44.0") found components: system unit_test_framework
  -- Performing Test HAVE_CJSON
  -- Performing Test HAVE_CJSON - Success
  -- Found cjson: /usr/include/cjson
  -- Performing Test QuadMath_COMPILES
  -- Performing Test QuadMath_COMPILES - Success
  -- Found QuadMath: (Supported by compiler)
  -- Found Python3: /usr/sbin/python3.12 (found version "3.12.3") found components: Interpreter Development.Module
  -- Performing Test HAS_FLTO
  -- Performing Test HAS_FLTO - Success
  -- Found pybind11: /usr/include (found version "2.12.0")
  -- Performing Test HAVE_ATTRIBUTE_ALWAYS_INLINE
  -- Performing Test HAVE_ATTRIBUTE_ALWAYS_INLINE - Success
  -- Performing Test HAS_ATTRIBUTE_UNUSED
  -- Performing Test HAS_ATTRIBUTE_UNUSED - Success
  -- Performing Test HAS_ATTRIBUTE_DEPRECATED
  -- Performing Test HAS_ATTRIBUTE_DEPRECATED - Success
  -- Performing Test HAS_ATTRIBUTE_DEPRECATED_MSG
  -- Performing Test HAS_ATTRIBUTE_DEPRECATED_MSG - Success
  -- CMake version: 3.29.3
  -- Linux distribution: Arch Linux
  -- Target architecture: x86_64
  -- Found Git: /usr/sbin/git (found version "2.45.0")
  fatal: not a git repository (or any of the parent directories): .git
  -- Source code repository: not found!
  -- GNU C++ compiler version: 14.1.1 20240507
  -- Linker: ld 2.42.0
  Processing opm_defaults opm-common
  -- Checking to see if CXX compiler accepts flag -Wl,--enable-new-dtags
  -- Checking to see if CXX compiler accepts flag -Wl,--enable-new-dtags - yes
  -- Precompiled headers: disabled
  -- Build type: None
  -- Checking to see if CXX compiler accepts flag -mtune=native
  -- Checking to see if CXX compiler accepts flag -mtune=native - yes
  -- Found OpenMP_C: -fopenmp (found version "4.5")
  -- Found OpenMP_CXX: -fopenmp (found version "4.5")
  -- Found OpenMP: TRUE (found version "4.5")
  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
  -- Found Threads: TRUE
  -- Checking to see if CXX compiler accepts flag -pthread
  -- Checking to see if CXX compiler accepts flag -pthread - yes
  -- Could NOT find CppCheck (missing: CPPCHECK_PROGRAM) 
  -- Disabling clang-check as CMAKE_EXPORT_COMPILE_COMMANDS is not enabled
  -- Could NOT find Valgrind (missing: VALGRIND_INCLUDE_DIR VALGRIND_PROGRAM) 
  -- Generating debug symbols: -ggdb3
  -- Looking for strip utility
  -- Looking for strip utility - found
  -- Performing Test HAVE_DYNAMIC_BOOST_TEST
  -- Performing Test HAVE_DYNAMIC_BOOST_TEST - Success
  -- Writing config file "/tmp/makepkg/opm-common/src/build-cmake/config.h"...
  -- This build defaults to installing in /usr
  -- Checking whether the file system is case-sensitive
  -- File system is case-sensitive
  -- Found Doxygen: /usr/sbin/doxygen (found version "1.10.0") found components: doxygen dot
  -- Writing version information to local header project-version.h
  -- Looking for sgemm_
  -- Looking for sgemm_ - not found
  -- Looking for sgemm_
  -- Looking for sgemm_ - found
  -- Found BLAS: /usr/lib/libblas.so
  -- Looking for cheev_
  -- Looking for cheev_ - not found
  -- Looking for cheev_
  -- Looking for cheev_ - found
  -- Found LAPACK: /usr/lib/liblapack.so;/usr/lib/libblas.so
  -- Performing Test HAVE_CXA_DEMANGLE
  -- Performing Test HAVE_CXA_DEMANGLE - Success
  -- Found MPI_C: /usr/lib/libmpi.so (found version "3.1")
  -- Found MPI_CXX: /usr/lib/libmpi.so (found version "3.1")
  -- Found MPI: TRUE (found version "3.1")
  -- Found GMP: /usr/lib/libgmpxx.so
  -- Found dune-common: /usr/include
  -- Looking for MPI_Finalized
  -- Looking for MPI_Finalized - found
  -- Version 2.9.1 of dune-common from /usr/lib/cmake/dune-common
  -- Configuring done (6.3s)
  -- Generating done (0.6s)
  -- Build files have been written to: /tmp/makepkg/opm-common/src/build-cmake
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/opm/common/utility/parameters/ParameterTools.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/opm/common/utility/numeric/calculateCellVol.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/opm/common/utility/numeric/MonotCubicInterpolator.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/opm/common/utility/numeric/RootFinders.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/common/Spline.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/common/Tabulated1DFunction.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/common/TridiagonalMatrix.cpp.o
  [ 26%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/common/UniformXTabulated2DFunction.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/components/CO2.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/components/H2.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/densead/Evaluation.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/fluidmatrixinteractions/EclEpsScalingPoints.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/fluidsystems/BlackOilFluidSystem.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/fluidsystems/blackoilpvt/DeadOilPvt.cpp.o
  [ 28%] Building CXX object CMakeFiles/opmcommon.dir/opm/material/fluidsystems/blackoilpvt/DryGasPvt.cpp.o
  In file included from /tmp/makepkg/opm-common/src/opm-common-release-2024.04-final/opm/material/densead/Evaluation.cpp:28:
  /tmp/makepkg/opm-common/src/opm-common-release-2024.04-final/opm/material/common/quad.hpp:50:7: error: redefinition of ‘class std::numeric_limits<__float128>’
     50 | class numeric_limits<quad>
        |       ^~~~~~~~~~~~~~~~~~~~
  In file included from /usr/include/c++/14.1.1/bits/specfun.h:44,
                   from /usr/include/c++/14.1.1/cmath:3898,
                   from /tmp/makepkg/opm-common/src/opm-common-release-2024.04-final/opm/material/common/quad.hpp:32:
  /usr/include/c++/14.1.1/limits:2089:12: note: previous definition of ‘class std::numeric_limits<__float128>’
   2089 |     struct numeric_limits<__float128>
        |            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  make[3]: *** [CMakeFiles/opmcommon.dir/build.make:3734: CMakeFiles/opmcommon.dir/opm/material/densead/Evaluation.cpp.o] Error 1
  make[3]: *** Waiting for unfinished jobs....
  In file included from /tmp/makepkg/opm-common/src/opm-common-release-2024.04-final/opm/material/components/CO2.cpp:27:
  /tmp/makepkg/opm-common/src/opm-common-release-2024.04-final/opm/material/common/quad.hpp:50:7: error: redefinition of ‘class std::numeric_limits<__float128>’
     50 | class numeric_limits<quad>
        |       ^~~~~~~~~~~~~~~~~~~~
  In file included from /usr/include/c++/14.1.1/bits/specfun.h:44,
                   from /usr/include/c++/14.1.1/cmath:3898,
                   from /tmp/makepkg/opm-common/src/opm-common-release-2024.04-final/opm/material/Constants.hpp:30,
                   from /tmp/makepkg/opm-common/src/opm-common-release-2024.04-final/opm/material/components/CO2.hpp:31,
                   from /tmp/makepkg/opm-common/src/opm-common-release-2024.04-final/opm/material/components/CO2.cpp:25:
  /usr/include/c++/14.1.1/limits:2089:12: note: previous definition of ‘class std::numeric_limits<__float128>’
   2089 |     struct numeric_limits<__float128>
        |            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  make[3]: *** [CMakeFiles/opmcommon.dir/build.make:3706: CMakeFiles/opmcommon.dir/opm/material/components/CO2.cpp.o] Error 1
  make[2]: *** [CMakeFiles/Makefile2:551: CMakeFiles/opmcommon.dir/all] Error 2
  make[1]: *** [CMakeFiles/Makefile2:7114: CMakeFiles/opmcommon_python.dir/rule] Error 2
  make: *** [Makefile:3286: opmcommon_python] Error 2
  ```
</details>

--- a/cmake/Modules/FindQuadMath.cmake
+++ b/cmake/Modules/FindQuadMath.cmake
@@ -46,4 +46,24 @@ if(QuadMath_FOUND AND NOT TARGET QuadMath::QuadMath)
   target_compile_options(QuadMath::QuadMath INTERFACE
     $<$<CXX_COMPILER_ID:GNU>:-fext-numeric-literals>
   )
+
+  # Check for numeric_limits specialization
+  cmake_push_check_state()
+  set(CMAKE_REQUIRED_LIBRARIES quadmath)
+  set(CMAKE_REQUIRED_FLAGS)
+  if(${CMAKE_CXX_COMPILER_ID} STREQUAL GNU)
+    set(CMAKE_REQUIRED_FLAGS "-fext-numeric-literals")
+  endif()
+  set(CMAKE_REQUIRED_FLAGS "${CMAKE_REQUIRED_FLAGS} -D_GLIBCXX_USE_FLOAT128")
+  check_cxx_source_compiles("
+  #include <limits>
+  int main()
+  {
+     static_assert(std::numeric_limits<__float128>::is_specialized);
+     return 0;
+  }" QuadMath_HAS_LIMITS)
+  cmake_pop_check_state()  # Reset CMAKE_REQUIRED_XXX variables
+  if(QuadMath_HAS_LIMITS)
+    target_compile_definitions(QuadMath::QuadMath INTERFACE LIMITS_HAS_QUAD=1)
+  endif()
 endif()

--- a/examples/make_ext_smry.cpp
+++ b/examples/make_ext_smry.cpp
@@ -17,7 +17,7 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-
+#include <algorithm>
 #include <filesystem>
 #include <iostream>
 #include <getopt.h>

--- a/opm/material/common/quad.hpp
+++ b/opm/material/common/quad.hpp
@@ -45,6 +45,7 @@ typedef __float128 quad;
 
 namespace std {
 
+#if !LIMITS_HAS_QUAD
 // provide the numeric limits for the quad precision type
 template <>
 class numeric_limits<quad>
@@ -97,6 +98,7 @@ public:
     static constexpr bool tinyness_before = std::numeric_limits<double>::tinyness_before;
     static constexpr float_round_style round_style = round_to_nearest;
 };
+#endif // LIMITS_HAS_QUAD
 
 // provide some type traits for the quadruple precision type
 template <>

